python/ray/data/context.py

```diff
@@ -204,6 +206,7 @@ def __init__(
         self.use_ray_tqdm = use_ray_tqdm
         self.use_legacy_iter_batches = use_legacy_iter_batches
         self.enable_progress_bars = enable_progress_bars
+        self.file_shuffler = file_shuffler
```

Comment:
nit, there is no need to add add this `__init__` argument. 

```
@property
def file_shuffler_cls(self):
     # import here to avoid cyclic dependencies.
     return DEFAULT_FILE_SHUFFLER;
   
```

Reply:
I guess so, we can change the whole __init__ later. Let's do a separate PR? Just follow other configs here.